PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -33,27 +35,71 @@
     >
     Open 
     </v-btn>
+    </v-row>
+
+     <v-row justify="center">
+      <div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataPersonalInfo.familyName}}, 
+					{{dataPersonalInfo.givenName}} {{dataPersonalInfo.middleName}} {{dataPersonalInfo.suffix}}</span></div>
+    </v-row>
+
 
+    <v-row justify="center">
+      <v-btn color="secondary" class="ma-2 white--text">
+								Download
+			</v-btn>
     </v-row>
+    
+ 
+
+<v-row justify="center">
+  <div class="font-weight-medium"><br><br>How do you want to receive this QR code?</div>
+</v-row>
+<v-row justify="center">
+  <v-col cols="3" sm="3" md="3">
```

Comment:
To make the format tablet and mobile friendly, this seems to be a helpful adjustment:
<v-col cols="8" sm="4" md="3" lg="3">